Momento/MomentoSigner.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.IdentityModel.Tokens.Jwt;
+using Microsoft.IdentityModel.Tokens;
+using MomentoSdk.Exceptions;
+
+namespace MomentoSdk
+{
+    public class MomentoSigner
+    {
+
+        /// <summary>
+        /// Create a pre-signed HTTPS URL to retrieve an object.
+        /// </summary>
+        /// <param name="hostname">Hostname of the SimpleCacheService. Use the value returned from CreateSigningKey's response.</param>
+        /// <param name="jwkJsonString">JWK in JSON string. Use the value returned from CreateSigningKey's response.</param>
+        /// <param name="cacheName">The name of the Cache.</param>
+        /// <param name="cacheKey">The key of the Object.</param>
+        /// <param name="expiryEpochSeconds">The timestamp that the pre-signed URL is valid until.</param>
+        /// <returns></returns>
+        public string CreatePresignedUrlForGet(string hostname, string jwkJsonString, string cacheName, string cacheKey, uint expiryEpochSeconds)
+        {
+            var payload = CommonJwtBody(cacheName, cacheKey, expiryEpochSeconds);
+            payload.Add("method", new string[] { "get" });
+
+            var encodedJwt = CreateJwtToken(jwkJsonString, payload);
+
+            return $"https://{hostname}/cache/get/{cacheName}/{cacheKey}?token={encodedJwt}";
+        }
```

Comment:
I think this should be at least 2 methods, `SignAccessToken()` and `CreatePresignedUrlForGet()`.

The sign method should return a JWS that can be used in our sdk to interact with Momento. I realize atm we bake the endpoint into the URL. We need to take it out, and when we do this access token will also work in the grpc direct client.

The CreatePresignedUrlForGet/Set method can call the SignAccessToken method and then simply do the string format on the result.

Reply:
Done.